src/plugins/vuetify.js

```diff
@@ -8,6 +8,7 @@ import NomadIcon from '@/components/Icons/Nomad.vue'
 import PrefectIcon from '@/components/Icons/Prefect.vue'
 import TwilioIcon from '@/components/Icons/twilio.vue'
 import PagerDutyIcon from '@/components/Icons/PagerDuty.vue'
+import newPagerDutyIcon from '@/components/Icons/newPagerDuty.vue'
```

Comment:
Is there a reason not to replace the existing pager duty icon?

Reply:
Yes - we're using the original pagerDuty logo in automations and cloud hooks.  It is a small PD.  This one is the full Pager Duty name we use as a header at the top of the pager duty integrations page.